exotica_python/scripts/example_aico.py

```diff
@@ -1,26 +1,17 @@
 #!/usr/bin/env python
```

Comment:
I think we should offer the old and new example instead of overwriting - i.e. an example_aico.py as currently is and then an exampe_aico_trajectory_generator.py with the new.